### ISSUE TITLE
Handle the challenges won response better

### ISSUE THREAD

**Issue by calum** [calum/alexa-drag-race-skill#14]
Alexa reads `undefined`

```
{
  "version": "1.0",
  "response": {
    "outputSpeech": {
      "ssml": "<speak> Thorgy Thor has won 0 main challenges and 0 mini challenges: undefined </speak>",
      "type": "SSML"
    },
    "speechletResponse": {
      "outputSpeech": {
        "ssml": "<speak> Thorgy Thor has won 0 main challenges and 0 mini challenges: undefined </speak>"
      },
      "shouldEndSession": true
    }
  },
  "sessionAttributes": {}
}
```

**Comment by calum** [comment]
fixing now

**Comment by calum** [comment]
closed by #20
